Tools/autotest/sim_vehicle.py

```diff
@@ -698,6 +698,17 @@ def start_mavproxy(opts, stuff):
     run_cmd_blocking("Run MavProxy", cmd, env=env)
     progress("MAVProxy exitted")
 
+def mkdir_p(dir):
+    '''like mkdir -p'''
+    if not dir:
+        return
+    if dir.endswith("/"):
+        mkdir_p(dir[:-1])
+        return
```

Comment:
this is taken from mavproxy , maybe I should make a credit (https://github.com/ArduPilot/MAVProxy/blob/b2452edc85c6d8c83cc258e3e6219ac9b1268675/MAVProxy/modules/lib/mp_util.py#L88)
The reason that this check is need is if I use , let's say "rover", as directory name using only dir[:-1] will create only "rove"